segway/_util.py

```diff
@@ -197,11 +197,25 @@ def get_col_index(chromosome, trackname):
 
 
 def get_label_color(label):
-    color = SCHEME[label % NUM_COLORS]
-
+    # Use hash_label to replicate hashing across subprocesses, as the Python
```

Comment:
Are these changes still necessary with the `make_label_colors` function now in `output.py`? I thought there was going to be no more hashing.

Reply:
Thanks for catching this! I thought I deleted the hash function in merging but apparently it made its way through.